board/boards/uno.h

```diff
@@ -145,9 +126,11 @@ void uno_set_can_mode(uint8_t mode){
   }
 }
 
-void uno_usb_power_mode_tick(uint32_t uptime){
-  UNUSED(uptime);
-  if(bootkick_timer != 0U){
+void uno_board_tick(bool ignition, bool usb_enum, bool heartbeat_seen) {
+  UNUSED(ignition);
+  UNUSED(usb_enum);
+  UNUSED(heartbeat_seen);
+  if (bootkick_timer != 0U) {
```

Comment:
might want to increase `BOOTKICK_TIME` since the tick is now called at 8Hz instead of 1

Reply:
moved to the 1Hz loop